.eslintrc

```diff
@@ -1,12 +1,7 @@
 {
-  "extends": ["standard"],
+  "extends": ["semistandard"],
```

Comment:
@danielfdsilva @ricardomestre just a heads up that we had actually done this on purpose to simplify the dependencies. Didn't seem like `semistandard` was adding enough to make it worth the extra dep.